spec/amber/extensions/http_spec.cr

```diff
@@ -0,0 +1,13 @@
+require "../../../spec_helper"
+
+module Amber::Extensions
+  describe HTTPServerContext do
+    it "client_ip should exist and be nil" do
```

Comment:
I don't think we need to test for the existence of a particular method. Crystal is compiled and guarantees the method exists for that class. Maybe just `"client_ip should be nil"` is enough?